packages/atomic/src/components/facets-v1/facet-value-box-color/facet-value-box-color.tsx

```diff
@@ -0,0 +1,40 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {FacetColorValueProps} from '../facet-common';
+
+export const FacetValueBoxColor: FunctionalComponent<FacetColorValueProps> = (
+  props,
+  children
+) => {
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+  const ariaLabel = props.i18n.t('facet-value', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+  });
+
+  return (
+    <li key={props.displayValue} class="h-32">
+      <button
+        part="value-box"
+        onClick={() => props.onClick()}
+        class={`value-box box-border w-full h-full flex flex-col items-center rounded text-on-background hover:border-primary-light focus:border-primary-light focus:outline-none py-2 px-1 ellipsed ${
+          props.isSelected ? 'border-primary border-2' : 'border border-neutral'
+        }`}
+        aria-pressed={props.isSelected.toString()}
+        aria-label={ariaLabel}
+      >
+        <div
+          part={`value-${props.partValue}`}
+          class="w-full h-1/2 bg-neutral-dark mx-4 mt-1 mb-3 rounded-md"
+        ></div>
+        {children}
+        <span
+          title={count}
+          part="value-count"
+          class="text-neutral-dark ellipsed with-parentheses w-full text-sm my-0.5"
+        >
+          {count}
+        </span>
+      </button>
+    </li>
```

Comment:
Would it be possible to reuse the `FacetValueBox` component? It seems like the div on line 25 is the main addition? Maybe something like:

```
FacetValueBoxColor:

<FacetValueBox>
    <div
        part={`value-${props.partValue}`}
        class="w-full h-1/2 bg-neutral-dark mx-4 mt-1 mb-3 rounded-md"
    ></div>
    {children}
</FacetValueBox>

```

Reply:
The styling for the `FacetValueBox` and the `ColorFacetBox` are different enough that I think they warrant being separate components. Same thing for the checkboxes.